Formula/openssh.rb

```diff
@@ -53,9 +56,49 @@ def install
     # potential to break scripts, so recreate it for now.
     # Debian have done the same thing.
     bin.install_symlink bin/"ssh" => "slogin"
+
+    # https://opensource.apple.com/source/OpenSSH/OpenSSH-209.30.4/com.openssh.sshd.sb
+    (buildpath/"org.openssh.sshd.sb").write <<-EOS.undent
+      ;; Copyright (c) 2008 Apple Inc.  All Rights reserved.
+      ;;
+      ;; sshd - profile for privilege separated children
+      ;;
+      ;; WARNING: The sandbox rules in this file currently constitute
+      ;; Apple System Private Interface and are subject to change at any time and
+      ;; without notice.
+      ;;
+
+      (version 1)
+
+      (deny default)
+
+      (allow file-chroot)
+      (allow file-read-metadata (literal "/var"))
+
+      (allow sysctl-read)
+      (allow mach-per-user-lookup)
+      (allow mach-lookup
+        (global-name "com.apple.system.notification_center")
+        (global-name "com.apple.system.opendirectoryd.libinfo")
+        (global-name "com.apple.system.opendirectoryd.libinfo") ;; duplicate name as a work-around for 19978803
+        (global-name "com.apple.system.logger"))
+    EOS
+    (etc/"ssh").install "org.openssh.sshd.sb"
   end
 
   test do
     assert_match "OpenSSH_", shell_output("#{bin}/ssh -V 2>&1")
+
+    begin
+      system sbin/"sshd", "-p", "8022"
+      sleep 2
+      assert_match "sshd", shell_output("lsof -i :8022")
+    ensure
+      # sshd keeps running even after a Process.kill so
+      # we have to lean on the system to handle it.
+      # The "-n" flag means only the newest sshd will be killed
+      # which should be the one we've just launched for testing.
+      system "/usr/bin/pkill", "-n", "sshd"
```

Comment:
Because of the `-n` flag it'll kill the most recently launched `sshd`. Unless someone fires up their own `sshd` in the few seconds between the test starting & ending here that _should_ not happen, but there is the potential our `sshd` fails to start properly in this test, in which case the system `sshd` might get caught. Could maybe wrap it more protectively or something, hmm.